### PR TITLE
fix mistake val test split

### PR DESCRIPTION
**Pull request by alexisdougha**
Hello,

Thank you for your contribution and for sharing the code, the study is very interesting.

When training the CNN models, the peptides in the validation set were fully included in the test set, leading to a biased model testing. I propose here this little change to fix the issue.

According to the README, "_The folder_ `supporting_experiments` _contains all necessary code and data for replicating the supporting findings_" but it seems to be empty. Could you add the code you are referring to?

Thank you for this.

### CODE REVIEW

**Comment by amp91** [comment]
Hello, 

Thank you for reaching out and pointing out that the order of two lines in the data splitting was swapped. This appears to have been a copy-paste error during the code formatting for public release.

We plan to add the code with supporting findings and will notify you once it has been added to the repository.

Thanks again!
